nanoFramework.Device.Bluetooth/GenericAttributeProfile/GattServiceUuids.cs

```diff
@@ -0,0 +1,171 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace nanoFramework.Device.Bluetooth.GenericAttributeProfile
+{
+    /// <summary>
+    /// Represents an enumeration of the most well known Service UUID values, and provides
+    /// convenience methods for working with GATT service UUIDs, and static properties
+    /// providing service UUIDs for common GATT services. To view a list of all Bluetooth
+    /// SIG-defined service UUIDs, see Bluetooth SIG-defined Service UUIDs.
+    /// </summary>
+    public static class GattServiceUuids
+    {
+        ////
```

Comment:
please remove the dead code of comment why it should be present

Reply:
These are standard UUIDs from the Bluetooth SIG standards. I had contemplated if these should be in this nanoFramework version of this project. Are they taking up too much space if present? You could easily add codes directly you self. 

GattServiceUuids.cs - Needs to be updated
GattCharacteristicUuids - Needs to be updated
GattDescriptorUuids -  Complete.

The assembly is not that big so will leave them in and updated commented out code.